### PR TITLE
Update board.h

### PR DESCRIPTION
**Pull request by system47**

### CODE REVIEW

**Review by aidanbush — APPROVED** [review]
Very helpful, improves performance greatly.

File: src/board.h [hunk 1]
```diff
@@ -3,6 +3,7 @@
 #include <stdint.h>
 #include <string.h>
 #include <stdint.h>
+#include <harambe.h>
```
Comment (adfyong) [inline src/board.h]: You forgot to commit `harambe.h` too

**Comment by reboss** [comment]
I agree, it also improves readability.  I was confused as to what exactly this repo did at first but now I fully understand its purpose. 5 stars!!

File: src/board.h [hunk 2]
```diff
@@ -32,4 +33,4 @@ struct board *makeBoard(string fileName){
 
 	}
 
-}
\ No newline at end of file
+}
```
Comment (adfyong) [inline src/board.h:37]: Thanks for the newline

**Review by adfyong — CHANGES_REQUESTED** [review]

**Review by adfyong — APPROVED** [review]
Thanks @system47 . It compiles on my machine now.

**Comment by system47** [comment]
Fixed errors on compile and addressed @adfyong's concerns

**Pull request closed without merging**